gnuplot_files/plot_memory_base.gp

```diff
@@ -20,8 +20,8 @@ set output "__OUT_FILENAME__"
 set style fill solid border -1
 set grid xtics
 set grid ytics
-set xtics 100000
-set yrange [0:100]
+set xtics 1000000
```

Comment:
If I remove this, then the x-axis range fits the data better, but the axis labels usually overlap and are difficult to read.